PseudoLocalizer.Core/PseudoLocalizer.Core.csproj

```diff
@@ -10,4 +10,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Title>PseudoLocalizer.Core</Title>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Karambolo.PO.Compact" Version="1.5.1" />
```

Comment:
If this isn't already, set the version to the minimum supported version for the code here. The global tool itself can reference the latest version. That way we wouldn't force a newer version on something referencing this assembly.

Reply:
I downgraded to 1.3.0, which will omit some recent bug fixes (but none that affect the unit tests). I couldn't work out how to install in the global tool without updating the version in the Core project :(